### ISSUE TITLE
Auto backup not running Samsung S7

### ISSUE THREAD

**Issue by andreaallred76** [jberkel/sms-backup-plus#925]
Before you report your issue, please consider perusing the [https://github.com/jberkel/sms-backup-plus#faq](FAQ) and searching the issues page to see if it has already been reported.

### Please specify the following:

* Android version: 8.0.0
* Phone model / brand: Samsung S7
* SMS Backup+ version installed: 1.5.11
* Messaging app: Downloaded 'Messages' app with blue icon

**Comment by andreaallred76** [comment]
I've been reading around and wondered if you've updated to use the new API calls for Oreo?
https://developer.android.com/about/versions/oreo/background

**Comment by andreaallred76** [comment]
I installed Tasker and tried to do a profile and task using the 3rd party integration as well as running the intent script. I can see the jobs running in the logs but it still doesn’t backup the messages. Only a manual trigger is working. Is there any other work around?

**Comment by andreaallred76** [comment]
I've also tried using 3 different messaging apps. Two called Messages and one called Pulse. Still won't backup without manual intervention.

**Comment by mivk** [comment]
Same problem on Samsung A6 (A600FN), with same Android version 8.0.0.

**Comment by mivk** [comment]
Duplicate of #899

**Comment by LegoChicken** [comment]
Automatic backup doesn't work on Moto G6 Andriod 8. Manual back works.

**Comment by egburr** [comment]
Android version: 9
Phone model / brand: Google Pixel XL (version 1)
SMS Backup+ version installed: 1.5.11
Messaging app: native 'Messages' app with blue icon

I'm having the same problem on my phone. It has apparently been a problem for a long while. I just had to replace my phone the middle of December, and that is what brought this to my attention. I've tried to follow the FAQ, but Android 9 does not seem to have a "battery optimization" settings option for apps. Manual backup works fine; only automatic backup just doesn't seem to even try. The only entries in the sms_backup_plus.log are are few sequences of:
-backup requested (manual)
-starting backup (manual)
-backing up
-backing up
-backup finished
with appropriately matching timestamps.
